sdcm/nemesis.py

```diff
@@ -232,6 +232,9 @@ class Nemesis:  # pylint: disable=too-many-instance-attributes,too-many-public-m
     delete_rows: bool = False  # A flag denotes a nemesis deletes partitions/rows, generating tombstones.
     zero_node_changes: bool = False
 
+    additional_configs: list[str] = None  # Configs required for running nemesis, used in job generation
+    additional_params: dict[str, str] = None  # Parameters required for jenkins pipelines, used in job generation
```

Comment:
we probably should document this in the code of the generator 
or maybe it's own separate markdown, of in the cli docs...

Reply:
Like which parameters are required for which nemesis or existence of this parameters in general?